test/opsem/gstruct.ll

```diff
@@ -0,0 +1,83 @@
+; RUN: %seabmc "%s" 2>&1 | %oc %s
+; CHECK: ^unsat$
+; ModuleID = '/tmp/sea-n9bqyN/gstruct.pp.ms.bc'
+source_filename = "test-globals/gstruct.c"
+target datalayout = "e-m:e-p:32:32-f64:32:64-f80:32-n8:16:32-S128"
+target triple = "i386-pc-linux-gnu"
+
+@a = internal unnamed_addr global [3 x i32] [i32 1, i32 2, i32 3], align 4
+@llvm.used = appending global [4 x i8*] [i8* bitcast (void ()* @seahorn.fail to i8*), i8* bitcast (void (i1)* @verifier.assume to i8*), i8* bitcast (void (i1)* @verifier.assume.not to i8*), i8* bitcast (void ()* @verifier.error to i8*)], section "llvm.metadata"
+
+declare i32 @unknown1(...) local_unnamed_addr #0
+
+declare void @verifier.assume(i1)
+
+declare void @verifier.assume.not(i1)
+
+declare void @seahorn.fail()
+
+; Function Attrs: noreturn
+declare void @verifier.error() #1
+
+declare void @seahorn.fn.enter() local_unnamed_addr
+
+; Function Attrs: nounwind
+define i32 @main() local_unnamed_addr #2 {
+entry:
+  tail call void @seahorn.fn.enter() #3
+  %0 = tail call i32 bitcast (i32 (...)* @unknown1 to i32 ()*)() #3
+  %1 = icmp eq i32 %0, 0
+  br i1 %1, label %._crit_edge, label %.lr.ph
+
+.lr.ph:                                           ; preds = %entry
+  br label %2
+
+; <label>:2:                                      ; preds = %.lr.ph, %2
+  %.0.i1 = phi i32 [ 1, %.lr.ph ], [ %6, %2 ]
+  %3 = load i32, i32* getelementptr inbounds ([3 x i32], [3 x i32]* @a, i32 0, i32 1), align 4, !tbaa !3
+  %4 = icmp eq i32 %3, 2
+  %5 = select i1 %4, i32 2, i32 1
+  %6 = add nsw i32 %5, %.0.i1
+  store i32 %6, i32* getelementptr inbounds ([3 x i32], [3 x i32]* @a, i32 0, i32 2), align 4, !tbaa !8
+  store i32 %6, i32* getelementptr inbounds ([3 x i32], [3 x i32]* @a, i32 0, i32 0), align 4, !tbaa !9
+  %7 = tail call i32 bitcast (i32 (...)* @unknown1 to i32 ()*)() #3
+  %8 = icmp eq i32 %7, 0
+  br i1 %8, label %._crit_edge.loopexit, label %2
+
+._crit_edge.loopexit:                             ; preds = %2
+  br label %._crit_edge
+
+._crit_edge:                                      ; preds = %._crit_edge.loopexit, %entry
+  %9 = load i32, i32* getelementptr inbounds ([3 x i32], [3 x i32]* @a, i32 0, i32 2), align 4, !tbaa !8
+  %10 = icmp sgt i32 %9, 0
```

Comment:
in this, and other tests, it seems that the expected value in the array is `0`. Would be nicer to use other values since there are many reasons under which a comparison against `0` might evaluate to `true/false` in an unexpected way. Why not check for `2` or `42` instead?